### PR TITLE
A000110,A008589 and A008587

### PR DESCRIPTION
**Pull request by s4mking**
Remove previous PR too many errors on the git history. To work on a proper base I choose to rebase evrything.
This PR is for A000110,A008589 and A008587

### CODE REVIEW

File: oeis.py [hunk 1]
```diff
@@ -247,6 +247,44 @@ def A001220(start: int = 0, limit: int = 2) -> Collection[int]:
     return sequence[start:]
 
 
+@oeis
+def A008587(start: int = 0, limit: int = 20) -> Collection[int]:
+    "Multiples of 5."
+    sequence = []
+
+    for i in range(start, limit):
```
Comment (JulienPalard) [inline oeis.py]: Beware, range expects `start`, `stop`, not `start`, `limit`. Also the `stop` argument of `range` is excluded, so `range(10, 12)` is `10, 11`.

File: tests/test_A008589.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+@given(integers())
+def test_A008589(start):
+    test = A008589(3, 40)
+    assert 7 in test
+    assert 14 in test
+    assert 21 in test
+    assert 28 in test
+    assert A008589(start, 1)[0] % 7 == 0
```
Comment (JulienPalard) [inline tests/test_A008589.py]: You should split this in two distinct tests, no need to test `A008589(3, 40)` for every `start`.

**Comment by JulienPalard** [comment]
ping.

**Comment by s4mking** [comment]
I don't understand why there is an error with an index out of range with this code : 
`@given(integers())
def test_A008589_null(start):
    assert A008589(start, 1)[0] % 7 == 0`

Here is what I got : 

`start = 1

    @given(integers())
    def test_A008589_null(start):
        print(start)
>       assert A008589(start, 1)[0] % 7 == 0
E       IndexError: list index out of range
`
Have I to catch if user send 1 to the start?
Thanks.

**Comment by JulienPalard** [comment]
You're mixing `limit` and `stop`. Your function when given `start=10, limit=10` should return `10` elements not zero.

**Comment by s4mking** [comment]
Ok  sorry I get it

**Comment by s4mking** [comment]
I don't understand this error, it doesn't seem to be relative to my PR  ? 
Capture d’écran 2019-12-29 à 09 51 08

**Comment by JulienPalard** [comment]
> I don't understand this error, it doesn't seem to be relative to my PR ?

It looks like it tells about an error in A000110. Look at the test code, it checks that:

    A000110(0, 20)[1:] == A000110(1, 19)

which should be right dependless on the sequence implemented.

**Comment by JulienPalard** [comment]
@s4mking thanks!

**Pull request merged**